tests/ui/thir-print/thir-tree-loop-match.stdout

```diff
@@ -121,7 +121,7 @@ body:
                                                                                                         span: $DIR/thir-tree-loop-match.rs:12:17: 12:21 (#0)
                                                                                                         kind: PatKind {
                                                                                                             Constant {
-                                                                                                                value: Ty(bool, true)
+                                                                                                                value: Leaf(0x01)
```

Comment:
I do not understand why this does not print the `ty` field...

Reply:
Oh, I found it. `compiler/rustc_mir_build/src/thir/print.rs` completely duplicates the debug impl for `Pat` and `PatKind`. Wut...